README.md

```diff
@@ -1,10 +1,10 @@
-# `kcp` provides a true multi-tenant Kubernetes control plane for workloads on many clusters
+# `kcp` provides a multi-tenant Kubernetes API service and a transparent multi-cluster workload service
```

Comment:
I think the difference between these two sentences is:

current: Get people excited about potential
proposed: Be very precise about the components

Generally at this stage in a project and prototype I value both - but prioritize the former rather than the latter.  Do you understand why I would argue for that prioritization, and there are counter arguments you'd make (such that the first sentence doesn't actually accomplish getting people excited)?

Reply:
Yes, I understand that prioritization.  What do you think of keeping the top heading as it is currently merged and doing the other changes in this PR?